test/utils/expects.js

```diff
@@ -0,0 +1,13 @@
+const { expect } = require('@oclif/test')
+/**
+ *
+ * @param {string|number} buffer
+ * @param {number} now
+ */
+function expectDateOnFile (buffer, now) {
+  const file = buffer?.toString()
+  expect(file.includes(now)).to.equal(true)
+}
+module.exports = {
+  expectDateOnFile
+}
```

Comment:
@luizAlbuquerqueJr, achei massa a ideia do arquivo, tô supreso que funciona mas ainda bem que funciona.
Só uma coisa: o nome da função deveria ser expectTimestamp**In**File, questão de inglês só.

Reply:
Fiquei surpreso também. Done